models/glif_psc_double_alpha.h

```diff
@@ -0,0 +1,481 @@
+/*
+ *  glif_psc_double_alpha.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef GLIF_PSC_DOUBLE_ALPHA_H
+#define GLIF_PSC_DOUBLE_ALPHA_H
+
+#include "archiving_node.h"
+#include "connection.h"
+#include "event.h"
+#include "nest_types.h"
+#include "ring_buffer.h"
+#include "universal_data_logger.h"
+
+#include "dictdatum.h"
+
+/* BeginUserDocs: integrate-and-fire, current-based
+
+Short description
++++++++++++++++++
+
+Current-based generalized leaky integrate-and-fire (GLIF) models
+(from the Allen Institute)
+
+Description
++++++++++++
+
+``glif_psc_double_alpha`` provides five generalized leaky integrate-and-fire
+(GLIF) models [1]_ with alpha-function shaped synaptic currents.
+Incoming spike events induce a postsynaptic change of current modeled
+by sum of two alpha functions [2]_. The alpha function is normalized such that an event
+of weight 1.0 results in a peak current of the fast component of the alpha function to
```

Comment:
sentences got mixed up here

Reply:
Hi clinssen,

Thank you for your review. Please see my attempt to fix the description (I just pushed the changes). Each receptor gets two alpha function inputs at different time constants. Because these are alpha functions, there is only one time constant for each (Rise = Decay). You can define multiple receptor types by making the parameters to be arrays, but each receptor has exactly two alpha functions.

I wasn't sure how the math notation works, so I put down an equation, but it may not render correctly. I appreciate it if you could point me to a reference for formatting specifications.